xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -380,7 +370,7 @@ private void tokenizeCreditCardRequest(final TokenConfiguration tokenConfigurati
         tokenizeCreditCard(tokenConfiguration, card, new NetworkHandler<TokenCreditCard>().setResultListener(new ResultListener<TokenCreditCard>() {
```

Comment:
Is the method `tokenizeCreditCardRequest` still used? Please clean up unused methods.